recipes/viennarna/meta.yaml

```diff
@@ -6,26 +6,33 @@ package:
   version: {{ version }}
 
 build:
-  skip: True # [osx]
-  number: 2
+  number: 3
   run_exports:
     # Ronny said VRNA is stable in x.x releases
     - {{ pin_subpackage('viennarna', max_pin='x.x') }}
 
 source:
   url: https://www.tbi.univie.ac.at/RNA/download/sourcecode/2_6_x/ViennaRNA-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    - remove-register.patch
 
 requirements:
   build:
     - make
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    # pin and install swig on osx due to some incompatibility with
+    # automatically installed swig
+    - swig ==4.1.1 # [osx]
```

Comment:
Thanks for the detailed comment. Is there an issue with using the latest version of swig from conda-forge? Just wondering if it should be pinned this specifically. Just want to make sure it's clear why it was done if it breaks down the road.

Reply:
It rather looked like a specific problem with swig 4.2.x maybe since the code was tested only with 4.1.1. I am not sure, whether e.g. 4.0.2 would work as well. Finally, I didn't pin the problem down to a specific version (pun intended;))---I didn't consider it a problem, since it is 'only' for build.  I will try to coordinate with the main author of the software Ronny Lorenz to see whether we can fix the code and ideally drop the pin.

Reply:
That sounds good to me. I'm fine with approving it the way it is. We've got the comment and this PR history as a breadcrumb trail. If the code upstream gets updated, then the recipe can be updated.